spec/javascripts/tests/MultiTableFilter_spec.js

```diff
@@ -0,0 +1,88 @@
+describe('filter table based on text field criteria', function () {
+  'use strict';
+
+  function visibleRows() {
+    return $('.js-filter-rows').find('tr:visible');
+  };
+
+  function setFilterText(text) {
+    $('.js-filter-input').val(text);
+    $('.js-filter-input').change();
+  };
+
+  describe('on a table', function() {
+    beforeEach(function (done) {
+      var self = this;
+
+      requirejs(['jquery', 'jqueryFastLiveFilter', 'MultiTableFilter'], function ($, jqueryFastLiveFilter, MultiTableFilter) {
```

Comment:
As @munckymagik pointed out, I was looking at the wrong attribute. You're completely right! We don't need `jqueryFastLiveFilter` here.